extension/data/guiobject/guiobject.css

```diff
@@ -1,12 +1,11 @@
 body {
```

Comment:
I would skip any visual glitches for now. It would be important to get this feature in and a new release of memchaser out. The old widget system will be removed soon in Nightly builds. So if there are specific issues can you please file them as new issues? Thanks.